aequilibrae/project/network/osm/osm_downloader.py

```diff
@@ -61,7 +64,7 @@ def doWork(self):
             msg = f"Downloading polygon {counter + 1} of {len(self.polygons)}"
             self.logger.info(msg)
             self.__emit_all(["Value", counter])
```

Comment:
This looks COMPLETELY different than other message-emitting. May be working on the QGIS side, but does NOT conform with the use for other messages

Reply:
All signals in *osm_downloader* and *osm_builder* are fixed. The necessary fixes in QGIS were also done.